client/flutter/lib/components/page_button.dart

```diff
@@ -28,7 +28,7 @@ class PageButton extends StatelessWidget {
     this.crossAxisAlignment = CrossAxisAlignment.start,
```

Comment:
If this file is solely a formatting change with 0 material modifications please revert.

Reply:
Cool, It had changes originally but the repo was updated.